core/src/main/java/org/elasticsearch/index/query/IndexQueryParserService.java

```diff
@@ -153,24 +153,6 @@ IndicesQueriesRegistry indicesQueriesRegistry() {
         return indicesQueriesRegistry;
     }
 
-    //norelease this needs to go away
-    public ParsedQuery parse(QueryBuilder queryBuilder) {
```

Comment:
What's the method of creating a ParsedQuery with this removed? In SearchService.parseSource() we need to create a ParsedQuery from the QueryBuilder and set it on the SearchContext. should we not just modify this method to call the toQuery() method on the queryBuilder and then create the ParsedQuery object?